app/assets/stylesheets/components/_va-crisis-line.scss

```diff
@@ -0,0 +1,158 @@
+// TODO: Remove !important when .footer a selector gets simplified
+
+.va-crisis-panel {
+  color: $color-primary-darkest;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  z-index: 2;
+  
+  a {
+    padding: 0;
+    background-image: none;
+  }
+}
+
+.va-crisis-panel-body {
+  background: $color-gray-lightest;
+  list-style: inherit;
+  margin: 0;
+  padding: 0;
+
+  p {
+    margin: 0;
+    padding: 2.5rem;
+    font-size: 1.4rem;
+  }
+
+  ul {
+    font-size: 1.7rem;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+  }
+}
+
+.va-crisis-panel--notouch {
+  bottom: 0;
+  right: 0; 
+  width: 33rem;
+  position: absolute;
+  
+  -webkit-transform: translateY(243px);
+      -ms-transform: translateY(243px);
+          transform: translateY(243px);
+  
+    -webkit-transition: transform 250ms;
+            transition: transform 250ms;
+           
+  li {
+    padding: 2.5rem 0 0;
+  }
+  
+  @media #{$small-only} {
+    display: none;
+  }
+}
+
+.va-crisis-panel--open {
+  -webkit-transform: translateY(0);
+      -ms-transform: translateY(0);
+          transform: translateY(0);
+}
+
+.va-crisis-line {
+  background-color: $color-gold;
+  color: rgb(0,0,0);
+  display: block;
+  font-weight: bold !important;
+  margin: 0 auto; 
+  padding: .25rem 0;
+  text-align: center;
+  text-decoration: none !important;
+  
+  span {
+    font-weight: bold;
+  }
+  
+  &:hover {
+    background-color: $color-gold;
+    border-bottom: none;
+    color: rgb(0,0,0);
+    text-decoration: none;
+  }
+  
+  @media #{$small-only} {
+    padding: .25em 0;
+  }
+}
+
+//===== VA CRISIS LINE BUTTONS 
+.va-crisis-line--touch {
+  border-radius: .2em;
+  margin: 2.5em 0;
+}
+
+.va-crisis-line--notouch {
+  background: $color-gold url(/assets/images/arrow-up.svg) no-repeat 97% center;
```

Comment:
@webinista This path (and the one below it) doesn't match the directory where the file is now. Will it automagically find it or is this a typo?